examples/serenity/hello_world/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "hello-world"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+
+[dependencies]
+serenity = { version = "0.11.5", default-features = false, features = ["client", "gateway", "rustls_backend", "model"] }
+shuttle-service = { version = "0.5.0", features = ["bot-serenity", "secrets", "sqlx-postgres"] }
+sqlx = { version = "0.6", features = ["runtime-tokio-native-tls", "postgres"] }
+tracing = "0.1.36"
```

Comment:
This should still be `log` until we completed the switch to tracing (else they will be lost if I'm remember correctly)

Reply:
It seems to work with the tracing Serenity uses under the hood, but I'll change it for now if there is some issue with this currently.